docs/INSTALL.md

```diff
@@ -57,12 +56,12 @@ Then build and install a local copy of libsecp256k1 for python-bitcointx:
 
     mkdir -p deps
     cd deps
-    git clone git@github.com:bitcoin-core/secp256k1
+    git clone https://github.com/bitcoin-core/secp256k1.git
     cd secp256k1
     git checkout 490022745164b56439688b0fc04f9bd43578e5c3
     make clean
     ./autogen.sh
-    ./configure --prefix JM_ROOT --enable-module-recovery --disable-jni --enable-experimental --enable-module-ecdh --enable-benchmark=no
```

Comment:
this is an unrecognized option
related https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1657